piwik/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM quay.io/continuouspipe/php7-apache
+
+MAINTAINER Alex Biddle <alex.biddle+cp-dockerfiles@inviqa.com>
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y php-geoip php-dev libgeoip-dev && apt-get clean
+
+RUN rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+RUN echo "extension=geoip.so" >> /etc/php/7.0/apache2/php.ini
+RUN echo "geoip.custom_directory=/app/misc" >> /etc/php/7.0/apache2/php.ini
+
+WORKDIR /app/web
+RUN curl -L -O https://builds.piwik.org/latest.tar.gz && \
+    tar --strip 1 -xzf latest.tar.gz && \
+    rm latest.tar.gz
+
+RUN chown -R www-data:www-data *
+RUN chmod -R 0755 tmp
+
+
+WORKDIR /app/misc
+RUN curl -L -O http://geolite.maxmind.com/download/geoip/database/GeoLiteCountry/GeoIP.dat.gz && gunzip GeoIP.dat.gz
+RUN curl -L -O http://geolite.maxmind.com/download/geoip/database/GeoLiteCity.dat.gz && gunzip GeoLiteCity.dat.gz && mv GeoLiteCity.dat GeoIPCity.dat
+RUN chmod a+w /app/misc
```

Comment:
This would be better as just being readable by the web server user, rather than writable too unless piwik has a built in auto-update mechanism for GeoIP.
Perhaps the geoip package provided by ubuntu and corresponding cronjob would be a better choice?